backend/src/main/java/com/thecheatsheet/cheatsheet/entity/CodeSegmentsEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.thecheatsheet.cheatsheet.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * @Author WYB
+ * @Date 2020/10/26
+ **/
+@Setter
+@Getter
+@ToString
+public class CodeSegmentsEntity {
+
+    String id;
+    String title;
+    String desc;
+    String code;
+    String language;
+    String createAt;
```

Comment:
Should not `createAt` and `lastModify` be `long` type, they are timestamps, which is usually a 64bit integer.

Reply:
indeed